cdc/sink/ddlsink/mysql/mysql_ddl_sink.go

```diff
@@ -84,6 +84,11 @@ func NewDDLSink(
 		return nil, err
 	}
 
+	cfg.IsTiDB, err = pmysql.CheckIsTiDB(ctx, db)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
If the downstream is MySQL, an error is returned here, so the changefeed cannot work properly ?

Reply:
MySQL downstream do not return an error